servicebus/data-plane/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Doesn't the license exist at the root?

Reply:
Will update the changes in one of next PR.